server/routerlicious/packages/gitresources/api-extractor.json

```diff
@@ -1,12 +1,12 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "@fluidframework/build-common/api-extractor-common-report.json",
-	"apiReport": {
-		"enabled": true,
-		"reportFolder": "<projectFolder>/../../api-report/"
-	},
-	"docModel": {
-		"enabled": true,
-		"apiJsonFilePath": "<projectFolder>/../../_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
+	"extends": "../../../../common/build/build-common/api-extractor-base.json",
+	"messages": {
+		"extractorMessageReporting": {
+			// TODO: Add missing documentation and remove this rule override
+			"ae-undocumented": {
+				"logLevel": "none"
+			}
+		}
```

Comment:
The new locations for the API report files doesn't affect something in CI? Theorizing: if the pipeline publishes them from where it expects them to be today, that step might start failing because it won't find anything, or maybe just not complain but publish an empty artifact.

Reply:
They shouldn't. The `.api.md` files aren't consumed by anything. They just exist for PR reviews.

Reply:
This PR doesn't change the output of the json metadata that our docs build uses. I'm also not sure if we publish anything for server anyways, though we probably should (internally at least, probably not to ff.com).

Reply:
We're [creating an artifact (_api-extractor-temp) with _something_](https://dev.azure.com/fluidframework/public/_build/results?buildId=216687&view=artifacts&pathAsName=false&type=publishedArtifacts) but not the md files so this change should be fine. I looked at the json files in the artifact... and don't quite understand what purpose they serve, to be honest :/.

Reply:
Ok the ones for client packages make more sense, the ones from server seem kind of empty.

Reply:
If I'm interpreting [this](https://github.com/microsoft/FluidFramework/blob/0158446144a598e7cbbb2b15a2c320b121feb7cd/build-tools/packages/build-tools/src/docStats/README.md#L4) correctly, with the changes for FF 2.0 and the move towards requiring docs on exported objects, we might get to a point where that artifact isn't very useful anymore.

Reply:
> We're [creating an artifact (_api-extractor-temp) with _something_](https://dev.azure.com/fluidframework/public/_build/results?buildId=216687&view=artifacts&pathAsName=false&type=publishedArtifacts) but not the md files so this change should be fine. I looked at the json files in the artifact... and don't quite understand what purpose they serve, to be honest :/.

These artifacts are what get consumed by our `api-markdown-documenter` to create API docs from code comments.

Reply:
> If I'm interpreting [this](https://github.com/microsoft/FluidFramework/blob/0158446144a598e7cbbb2b15a2c320b121feb7cd/build-tools/packages/build-tools/src/docStats/README.md#L4) correctly, with the changes for FF 2.0 and the move towards requiring docs on exported objects, we might get to a point where that artifact isn't very useful anymore.

The `api.md` artifacts can be a useful tool for looking at the complete shape of a package's API. But as we move towards more build failures and fewer comments logged to the reports, they may serve less purpose in PR reviews. Though they will offer context for release scope changes over time (e.g. `@beta` -> `@public` change to one or more APIs in a PR).